test/parallel/test-tls-delayed-attach.js

```diff
@@ -24,17 +24,18 @@ const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
+
```

Comment:
Nit - it would be nice not to have any unrelated changes and if this could be changed back again.